app/scripts/controllers/provisionedService.js

```diff
@@ -0,0 +1,190 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:ProvisionedServiceController
+ * @description
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('ProvisionedServiceController', function ($scope,
+                                             $routeParams,
+                                             DataService,
+                                             ProjectsService,
+                                             $filter) {
+    $scope.projectName = $routeParams.project;
```

Comment:
It appears used by the "overview" navigation, when removed the project name becomes "undefined" and the navigation redirects towards an error page